### PR TITLE
Add range to gist:description

### LINKED ISSUE #1093: Add range of `xsd:string` to `gist:description`?
The datatype property `gist:description` does not currently have a range. `xsd:string` seems appropriate.

### PR DESCRIPTION
**Pull request by kstudzin**
Closes #1093

### CODE REVIEW

**Review by rjyounes — CHANGES_REQUESTED** [review]
Just need a release note.

**Review by uscholdm — COMMENTED** [review]
I suggest a scope note along these lines:

- Generally this property will only be used for data instances. Use `skos:defintion` for ontology concepts as well as categories that help to define the subject matter of the ontology.

File: docs/release_notes/issue1093-description-range.md [hunk 1]
```diff
@@ -0,0 +1 @@
+gist:description now has a range of xsd:string
```
Comment (rjyounes) [inline docs/release_notes/issue1093-description-range.md:1]: Please see guidance on formatting release notes.

**Review by rjyounes — CHANGES_REQUESTED** [review]

**Comment by rjyounes** [comment]
I like Michael's suggestion above for a scope note.

**Comment by kstudzin** [comment]
Okay - do you see this is as a replacement of the current scope note or an addition?

File: docs/release_notes/issue1093-description-range.md (hunk 1 above)
Comment (rjyounes) [inline docs/release_notes/issue1093-description-range.md:1]:
```suggestion
### Minor Updates

- Added range `xsd:string` to `gist:description`. Issue [#1093](https://github.com/semanticarts/gist/issues/1093).
```

**Review by rjyounes — CHANGES_REQUESTED** [review]

**Review by rjyounes — APPROVED** [review]

**Pull request merged**